api/.gitallowed

```diff
@@ -0,0 +1 @@
+AIzaSyCTI4Xz-UW_G2Q2RfknhcfdAnTHq5X5XuI
```

Comment:
what is this file for?

Reply:
This was to allowlist the API key everywhere, because I keep getting errors from git. However, this doesn't seem to solve the problem either, so I will probably revert it.